FreshRSS/FreshRSS
# Fix: show password with touch

![grafik](https://user-images.githubusercontent.com/1645099/164947603-764e615e-3d9a-4696-a5db-b665acb65deb.png)


The issue was before:
The show password button did not work with touch events


Changes proposed in this pull request:

- show password buttons works now with touch event
- had to change the behavior of the show password button. Before: show password while the buttons keep pressed. After: Click on button = show password. Click on the button again = hide the password. After 5 seconds the password will be hidden automatically


How to test the feature manually:

1. go to login page
2. type some password
3. click on the `show password` button
4. the password is shown
5. after 5 seconds the password will be hidden or click again on the `show password` button
6. the password text is hidden

Check it on "desktop" device (without touch, f.e. with a mouse) and on mobile device (with touch)

Pull request checklist:
